### ISSUE TITLE
Pluggin not useable from AssetLib install.

### ISSUE THREAD

**Issue by greenfox1505** [codecat/godot-tbloader#59]
The install from AssetLib seems to need content from `tb-gameconfig`, but it isn't included. Installing from the AssetLib should be everything you need to start a project, and nothing else.

**Comment by codecat** [comment]
True, this is currently not included because the folder is not a part of `addons/tbloader`. Hopefully we also won't need to do that at all when #4 is implemented, but I think including the files should be okay.

Do other assetlib packages include additional files outside of the `addons` folder? Is this an acceptable way to do it?

**Comment by greenfox1505** [comment]
There doesn't actually seem to be a ton of consistency. Some include files outside the `addons`, but I find those really annoying. Especially when they include their entire repo, including their `README.md` and it overwrites my own. The AssetLib is a kinda crapshoot of inconstancy. But there are definitely some "nicer" to use addons that follow some better consistency. Godot probably needs better defined standards here.

(IMHO) The more polished AssetLib installs just include everything you in their `addons/$NAME` directory. `addons/$NAME/README.md` is the "how to use" tutorial. Some include their examples, and I have mixed feelings about that (easy to find, but balloons your repo). If you need to anything that isn't immediately in the payload, it should be clearly outlined in THAT `addons/$NAME/README.md`. TBLoader is a addon that depends on an external application; every place TBLoader+TrenchBroom interact (on a user level) should be clearly tutorialized in that `README.md`.

I was not able to get TBLoader working last night, partly because it took a bit to even find this repo. Then I couldn't figure out how to or if I'm configuring the TBLoader+TrenchBroom properly. I'm still getting segfaults crashing Godot 4 (beta9) completely. Documentation could be better. If I get it working, I'll probably have to do a write up for my artists. I can post that if it's helpful. I'd like to get this working for GodotWildJam kicking off today, but I'll switch to Godot3+Qodot if I don't get it working.

**Comment by codecat** [comment]
I suppose it makes sense to keep those files and a little documentation for it in asset lib. Thanks for the input!

TBLoader currently only supports the latest version of Godot 4, which is beta 11. If you're keeping TBLoader updated, you should also keep Godot 4 updated!

**Comment by greenfox1505** [comment]
`Make sure you're using the latest version of Godot4` or better yet `This version of TBLoader was built for Godot4 beta11` would be nice to also have in the AssetLib documentation too.

**Comment by codecat** [comment]
I have made some changes to the asset lib package with 0.14.0, you can view the diff here: https://git.mrag.nl/nimble/tbloader-addonlib/commit/4eedba212f5d221d0d12c8f491bd7ea73ca72ba5

Hopefully this should be a little bit more of the necessary info for people to get started directly from asset lib. Let me know if I missed anything!
